src/PersonReader.java

```diff
@@ -1,6 +1,66 @@
-public class PersonReader {
-    public static void main(String[] args) {
-        // Write your code here ... ⬇
 
+    import java.awt.*;
+    import java.util.Scanner;
+    import javax.swing.JFileChooser;
+    import java.io.FileNotFoundException;
+    import java.io.File;
+    import java.util.StringTokenizer;
+
+
+    public class PersonReader
+{
+
+
+
+    public static void main(String[] args)
+    {
+
+        boolean hasFile = false;
+        File selectedFile = new File("");
+        JFileChooser choose = new JFileChooser();
+        String filename = "classmates.csv";
+        String lines = "";
+        String currToken = "";
+
+
+
+        choose.setCurrentDirectory(new File("C:\\Users\\winso\\OneDrive\\Documents\\UC"));
+
+        do {
+            if (choose.showOpenDialog((Component)null) == 0) {
```

Comment:
the file chooser function has this static property `JFileChooser.APPROVE_OPTION` that you could use instead of the `0` to improve readability 
```suggestion
            if (choose.showOpenDialog((Component)null) == JFileChooser.APPROVE_OPTION) {
```